chapter_appendix-tools-for-deep-learning/jupyter.md

```diff
@@ -115,12 +115,25 @@ The above is the address of the remote server `myserver`. Then we can use http:/
 
 We can use the `ExecuteTime` plugin to time the execution of each code cell in a Jupyter Notebook. Use the following commands to install the plugin:
 
-```
+```sh
 pip install jupyter_contrib_nbextensions
 jupyter contrib nbextension install --user
 jupyter nbextension enable execute_time/ExecuteTime
 ```
 
+### Hinter (without using `tab`)
```

Comment:
The english of this section isn't very smooth. I'll suggest to move it out from this PR and open another PR for it. @astonzhang Also I'll suggest to move all developer related sections to a separate doc.

Reply:
I don't think that we need to move these to a separate doc. We don't have too many developers. I'm new and bad at English.